share/github-backup-utils/ghe-backup-config

```diff
@@ -22,6 +22,10 @@ BACKUP_UTILS_VERSION="$(cat $GHE_BACKUP_ROOT/share/github-backup-utils/version)"
 
 # Add the bin and share/github-backup-utils dirs to PATH
 PATH="$GHE_BACKUP_ROOT/bin:$GHE_BACKUP_ROOT/share/github-backup-utils:$PATH"
+# In Solaris 11 reassure that gnu utils are used and add /usr/sbin for unlink
+if [ "SunOS 5.11" = "$(uname -sr)" ]; then
+  PATH="/usr/gnu/bin:$PATH:/usr/sbin"
+fi
```

Comment:
Having this here sets a precedent for similar behaviour and operations on other operating systems which we don't want.  This could turn into a messy collection of OS-specific `if`s.

I think this is better suited to live within your own copy of `backup.config` as this is the location best suited for customer/site specific customisations; you might have the GNU utils at `/usr/gnu/bin` on your Solaris 11 system, but it doesn't mean everyone does (the various GNU pkg are optional IIRC).

How about adding a commented out `PATH=` entry and generic comment to the end of the `backup.config-example` instead?